dashboard/test/ui/features/teacher_homepage.feature

```diff
@@ -62,3 +62,28 @@ Feature: Using the teacher homepage sections feature
     And the href of selector "a:contains(View Unit Overview)" contains the section id
     And I wait until element ".header_popup_body .uitest-ProgressBubble:first" is visible
     And the href of selector ".header_popup_body .uitest-ProgressBubble:first" contains the section id
+
+  Scenario: Navigate to course pages with course versions enabled
+    Given I am on "http://studio.code.org/home?enableExperiments=courseVersions"
+    When I see the section set up box
+    # By default, the '18-'19 version should be selected.
```

Comment:
Will these UI tests need to be updated as we add new script versions? Anything we can do to make it more evergreen?

Reply:
yes, these would need to be updated when we add new course versions. it's hard to say whether we should automate this -- it seems important to test what the user actually sees, and when we go to make version `'19-'20` visible and stable, it seems ok to have to update a few UI tests. If we do want to future-proof this, an approach for doing so isn't jumping out at me but I'm open to ideas.